javalib/src/main/scala/java/lang/System.scala

```diff
@@ -133,7 +133,8 @@ object System {
   def setErr(err: PrintStream): Unit =
     this.err = err
 
-  def gc(): Unit = GC.collect()
```

Comment:
I reviewed the documentation and can't see a reason for this change.

Reply:
Looks like it needs a `scalafmt` as well.

Reply:
> I reviewed the documentation and can't see a reason for this change.

The reason for this is to make sure that both methods are always in sync (and from the documentation, `Runtime.getRuntime().gc()` seems to be the authoritary method).
Who knows, maybe in the future there's a `GC.collectEden()` or a `GC.collectFull()` and only one of the methods gets updated?
Not that I think anyone will change in the near future, but doing it this way seemed a bit more future proof (and I don't think having an extra function call will be a bottleneck when forcing the GC)

> Looks like it needs a `scalafmt` as well.

Are you saying because of the `check-lint`? I think it failed with a `scripts/.coursier: line 1: `<h1>Internal Server Error</h1>'` (if not, maybe I messed up somewhere, but I think I reformatted both files)

Reply:
We have a `scripts/scalafmt` you can run.

It is a little unclear - the is that hint in the javadoc - https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#gc() I think rather than create and instance of `Runtime` maybe `System.gc()` can be the reference and call the GC and the Runtime can call that method. This seems simpler and avoids the allocation. If you use the `Runtime` method, you have to get the instance. WDYT?

Reply:
Makes sense, reversed the logic in ee52dec

Reply:
Thanks for this PR :-)